source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -185,11 +187,34 @@ class Editor(ia2Web.Editor):
 	TextInfo = EditorTextInfo
 
 
+class RedirectDocument(ia2Web.Ia2Web):
+	"""The NVDAObject for the redirect document object in Electron."""
+
+	def objectFromPointRedirect(self, x: int, y: int) -> IAccessible | None:
+		docObj: Document = self.previous.lastChild
+		redirect = docObj.IAccessibleObject.accHitTest(x, y)
+
+		if not redirect:
+			return None
+
+		redirect = IAccessibleHandler.normalizeIAccessible(redirect)
+		obj: IAccessible = IAccessible(IAccessibleObject=redirect, IAccessibleChildID=winUser.CHILDID_SELF)
+
+		return obj
+
+
 def findExtraOverlayClasses(obj, clsList):
 	"""Determine the most appropriate class(es) for Chromium objects.
 	This works similarly to L{NVDAObjects.NVDAObject.findOverlayClasses} except that it never calls any other findOverlayClasses method.
 	"""
 	if (
+		obj.role == controlTypes.Role.PANE
+		and obj.previous
+		and obj.previous.lastChild
+		and obj.previous.lastChild.windowClassName == "Chrome_RenderWidgetHostHWND"
```

Comment:
this seems fragile and may include other documents unexpectedly

Reply:
Do you have any suggestions?
Adding more constraints like before? Does this add more overhead?

I haven't encountered any problems with it in practice, at least so far.

Reply:
Ideally this would have an ID we could query to positively identify it.
Or even better, would just get fixed on the Electron side

Reply:
https://github.com/nvaccess/nvda/issues/17108#issuecomment-2334213761

Unfortunately, I'm afraid that won't work.
In the comment pointed to by the link above, I mentioned that there are differences in the mouse objects that prevent the use of 'Chrome_WidgetWin_1' as a judgment condition.